src/cpp/divergence.h

```diff
@@ -1,11 +1,29 @@
-/**
+/*
```

Comment:
You start with a ```/**``` here, and a ```/*``` for line two, then the SPDX. Other files start with a ```/*``` on line one and two blank lines lines. Pick one, and be consistent across all files. (see gradient.cpp, mixedbc.cpp, utils.cpp )

Reply:
Hi Jared - 

I have changed the files to be uniform and be consistent

Reply:
Nitpicking here, the top lines are not consistent. Here you have ```/* ``` then the SPDX license identifier.

In other files, you have ```/*``` then a blank line, then the SPDX info.

Reply:
Some files are still different than each other.
divergence.cpp and divergence.h are just two examples.
there is a difference between files starting as 
```
/*
 * 
 * SPDX-License-Identifier: GPL-3.0-only 
```
and
```
/*
 * SPDX-License-Identifier: GPL-3.0-only 
```